### PR TITLE
Display query result at editor footer

### PR DESCRIPTION
**Pull request by abbyhu2000**
### Description

Show query error.

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/query_enhancements/server/routes/index.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { ISearchStrategy } from '../../../data/server';
 import { API, SEARCH_STRATEGY } from '../../common';
 import { registerQueryAssistRoutes } from './query_assist';
 import { registerDataSourceConnectionsRoutes } from './data_source_connection';
+import { instance } from '../../../console/public/application/contexts/editor_context/editor_registry';
```
Comment (ashwin-pc) [inline src/plugins/query_enhancements/server/routes/index.ts:18]: think you accidentally included this import

File: yarn.lock
Comment (ashwin-pc) [inline yarn.lock:1]: dont commit this file

File: src/plugins/navigation/public/top_nav_menu/top_nav_menu.tsx [hunk 2]
```diff
@@ -156,6 +160,7 @@ export function TopNavMenu(props: TopNavMenuProps): ReactElement | null {
       <SearchBar
         {...searchBarProps}
         showDatePicker={![TopNavMenuItemRenderType.OMITTED, false].includes(showDatePicker!)}
+        data$={props.data$}
```
Comment (ashwin-pc) [inline src/plugins/navigation/public/top_nav_menu/top_nav_menu.tsx:163]: Dont pass the whole data observable as a prop. Just pass the query status as a prop. this might be the reason for the loading state issue that you are seeing

File: yarn.lock [hunk 3]
```diff
@@ -15986,7 +15986,7 @@ string-similarity@^4.0.1:
   resolved "https://registry.yarnpkg.com/string-similarity/-/string-similarity-4.0.4.tgz#42d01ab0b34660ea8a018da8f56a3309bb8b2a5b"
   integrity sha512-/q/8Q4Bl4ZKAPjj8WerIBJWALKkaPRfrvhfF8k/B23i4nzrlRj2/go1m90In7nG/3XDSbOo0+pu6RvCTM9RGMQ==
 
-"string-width-cjs@npm:string-width@^4.2.0":
+"string-width-cjs@npm:string-width@^4.2.0", string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.3:
```
Comment (AMoo-Miki) [inline yarn.lock:15989]: Please undo these changes to the lock file.
